numpy/__init__.pyi

```diff
@@ -326,6 +326,7 @@ from numpy.core.shape_base import (
     atleast_1d as atleast_1d,
     atleast_2d as atleast_2d,
     atleast_3d as atleast_3d,
+    atleast_3d as atleast_nd,
```

Comment:
```suggestion
    atleast_nd as atleast_nd,
```
Found the issue.

Reply:
Fyi, `from x import y as y` statement are used for re-exporting,
_i.e._ it tells type checkers that `y` can be imported both from `x` as well as the module in question. 

This doesn't work with `from x import y as z`-type statements though.

Reply:
Yes. I really have been staring at this for too long. As always, thank you for the catch!